starknet_py/tests/e2e/client_devnet/fixtures/contracts.py

```diff
@@ -0,0 +1,26 @@
+import pytest_asyncio
+
+from starknet_py.contract import Contract
+from starknet_py.tests.e2e.fixtures.contracts import deploy_v1_contract
+from starknet_py.tests.e2e.fixtures.contracts_v1 import declare_cairo1_contract
+from starknet_py.tests.e2e.fixtures.misc import load_contract
+
+
+@pytest_asyncio.fixture(scope="package", name="f_string_contract_class_hash")
```

Comment:
What's the reason to use `f_` prefix?
```suggestion
@pytest_asyncio.fixture(scope="package", name="string_contract_class_hash")
```

Reply:
I added prefix `f_` to distinguish class hash from forked devnet and "normal" devnet, because this contract has already existed there. Name "string_contract_class_hash" creates conflict with this [fixture](https://github.com/software-mansion/starknet.py/blob/development/starknet_py/tests/e2e/fixtures/contracts_v1.py#L136)

Reply:
I can't see any possibility of reusing this fixture. It is declared on other instance of devnet also with account bonded to that devnet.

Reply:
But I guess that if we use a devnet instance in fork mode for all tests, it would be possible to use the same fixture here?